### DISCUSSION TITLE
Customize Search trigger position on Notebook layout

### DISCUSSION THREAD

**Discussion by mnove** [fuma-nama/fumadocs#1163] [Q&A] [answered]
When I activate the notebook layout, the search is automatically displayed before the Navbar links. Is there a way to customise the search trigger with `fumadocs-ui/layouts/notebook`

I'd like to keep Search trigger in the sidebar, and only show the Navbar links on the navbar itself. Thanks!

**Comment by fuma-nama** [comment, marked as answer]
For further customisation, I would recommend to install the layout with Fumadocs CLI: https://fumadocs.vercel.app/docs/cli

```bash
pnpm fumadocs add
```

and add the layout you want to customise

**Reply by mnove** [reply]
Thanks, will look into it

**Comment by andrepatt** [comment]
Hi @mnove, did you manage to customize the navbar? Could you help or address me on how to do that? Thanks!

**Reply by mnove** [reply]
Unfortunately, I needed a navbar that was a little different from the default. I ended up creating my own navbar components and docs layout. 

I checked the source code of fumadocs to get an idea. Then I built my custom version. 
Maybe [check the notebook component](https://github.com/fuma-nama/fumadocs/blob/dev/packages/ui/src/layouts/notebook.tsx) and start from there. Or, as mentioned earlier in the discussion, you can use the CLI and install the layout as needed, then modify it.
